src/lib/registration/widgets/registration_code.rb

```diff
@@ -61,20 +61,19 @@ def store
       def register
         if skip?
           log.info("Empty value, skipping registration")
-          return true
+          return false
```

Comment:
Basically be consistent with the API only returns true if the registration success, I modified it based on test that I'm written, I thought that is more consistent and expected.